flutter/alds/lib/mappage.dart

```diff
@@ -27,112 +27,92 @@ class AldsMapPage extends StatefulWidget {
 }
 
 class _AldsMapPageState extends State<AldsMapPage> {
-  // State variables
   String _curLocationText = "";
   Position? _curPosition;
   final TextEditingController _controller = TextEditingController();
   String? _selectedLocation;
   List<String> locations = [];
   late bool _isLoading;
-
-  _AldsMapPageState();
```

Comment:
I think we should keep the constructor, though it's empty, to match standard Flutter conventions.